mingw-w64-git/PKGBUILD

```diff
@@ -67,6 +69,9 @@ build() {
 	compat/win32/git-wrapper.o git.res -lshlwapi &&
   gcc -mwindows -s -O2 -Wall -o cmd/gitk.exe gitk.res \
 	compat/win32/git-wrapper.o git.res -lshlwapi
+
+  # git-credential-wincred.exe
+  CC=gcc make -C contrib/credential/wincred
```

Comment:
I tried this without the CC=gcc, but for some reason `make` was defaulting to `cc` rather than `gcc`:
```
cc -march=x86-64 -mtune=generic -O2 -pipe -D_FORTIFY_SOURCE=2 -D__USE_MINGW_ANSI_STDIO=1 -pipe  git-credential-wincred.c   -o git-credential-wincred.exe
make: cc: Command not found
```
Seemed strange especially when I looked at the [Makefile](https://github.com/git-for-windows/git/blob/master/contrib/credential/wincred/Makefile). If there's a better fix let me know.